apps/prairielearn/assets/scripts/instructorRequestCourseLti13.ts

```diff
@@ -0,0 +1,35 @@
+import { decodeData, onDocumentReady } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  const courseRequestLti13Info = decodeData('courseRequestLti13Info');
+
+  if (courseRequestLti13Info === null) {
+    return;
+  }
+
+  const Lti13Modal = new bootstrap.Modal(document.getElementById('lti13FillModal'), {});
```

Comment:
This fails typechecking. I'm not sure the right way to reference bootstrap in an asset.

Reply:
You'd want `window.bootstrap...`. Regardless, this is incompatible with BS4. You'll need to use the jQuery API for both forwards and backwards compatibility until we no longer have the BS4 opt-in feature flag.

Reply:
Make sense. Switched to jQuery in 0a22979fe